specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1597,6 +1597,85 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/apis/sql/databases/{databaseName}/settings/throughput": {
```

Comment:
A few follow up questions:
1) Does "throughput" need to be independently RBAC'd from other settings? Would it make more sense to have a /settings/default resource type that can be amended later with other settings properties?
2) While the segmentation is legitimate, it can be cleaner and/or more flexible to model the static path parameters (e.g. "sql" in the above) as enum parameters, see here: https://swagger.io/docs/specification/describing-parameters/

Reply:
Thanks for your reminder. For our case, all the settings have the same RBAC